src/feedback/data/sagas.js

```diff
@@ -24,18 +24,6 @@ export function* handleErrors(e, clearExistingMessages) {
       yield put(addMessage(message.code, message.userMessage, message.data, message.messageType));
     }
   }
-  if (e.fieldErrors !== undefined) {
```

Comment:
This is dead code?

Reply:
I would have expected this to still be used for field-specific validation error messages - as I recall it's how we communicate server-side validation of model fields back to the client.

Reply:
Or are we saying we no longer want to display field errors at the top of the page?  I believe that was an a11y requirement.

Reply:
So, when I exposed my new errors (on the CC fields) using this method, they show up as banners and on the actual fields. When I trigger the same errors on the current payment page (not using the new validation inside `payment/sagas.js`, the messages only show up by the specific field that had an error. So, either we should be putting everything in banners, or we should be putting them only on fields, but they should be consistent.

Reply:
I contest that it's unnecessary, and dumps duplicates of the error messages that are on each field onto the top of the page. But, @davidjoy suggests maybe there's an a11y issue (in which case, the other validation flow is violating that a11y consideration).